### PR TITLE
add lc scatter as fit param + new residuals distribution plot + minor plot fixes

### PR DESCRIPTION
**Pull request by cpiaulet**
- new fitting parameter: scatter_ppm
- parameter used for chi squared and log likelihood calculations
- make code robust to hitting upper or lower bound in initial lsq
- add a new plot: distribution of residuals normalized by the single-point scatter in the light curve
- fix xlabel for lc plot
- fix uncertainties in residuals panel following lc fit

### CODE REVIEW

**Comment by taylorbell57** [comment]
Have you tested whether the new scatter_ppm parameter works well with shared fits? I've yet to do any testing with the new parameter, but I can confirm that the code behaves as before if you don't choose to fit for it

**Review by taylorbell57 — CHANGES_REQUESTED** [review]
As of right now this breaks for shared fits, but I have confirmed it works well for unshared fits. Let me know if you would like help getting it to work

**Comment by cpiaulet** [comment]
I have never ran shared fits with Eureka, would love some help with at least testing so I understand how the shared fits are coded and where it breaks :)

**Comment by taylorbell57** [comment]
You could run S4 with more than one wavelength bin (can just do two to make things quick to test) and then set any parameter in the S5 param ecf to 'shared' instead of 'free' (e.g. t0 is a good one since the transit timing should be wavelength independent). I tried it once and got an array shape error when evaluating the new model and it wasn't immediately clear to me why model.eval() would give the wrong shape so I just left a review here for now since I have to referee a paper and write a reference letter before the end of the week. Hopefully it'll be an easy fix for you, otherwise I (in a day or few) or someone else can help if you get stuck

**Comment by taylorbell57** [comment]
Then again I just tried a shared fit without scatter_ppm and I get the same error, so maybe it is caused by my changes rather than yours or yours breaks regardless of whether you fit scatter_ppm in a shared fit

**Comment by taylorbell57** [comment]
Well my issue may or may not be related to your PR, but I can already tell it won't work with shared fits as the different scatter_ppm parameters that get made for each wavelength don't get applied to each wavelength's lightcurve. Instead the first one gets used for all wavelengths and all the others are only constrained by the priors and are unused.

**Comment by taylorbell57** [comment]
@cpiaulet this PR ended up getting merged in with mine which is fair enough since your edits don't really break the code for shared fits any more than it already was. Have you made any progress on getting the scatter_ppm to work with shared fits?
